src/adagrams.js

```diff
@@ -1,15 +1,153 @@
+/*constructing a leeterpool object with letter as key and no. of times it can be drawn as value*/
+const LETTERPOOL = {
```

Comment:
Nice job using capital letters to convey to other devs the value is fixed. We separate words with `_`, like:

```suggestion
const LETTER_POOL = {
```